backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/DefaultAndroidInput.java

```diff
@@ -359,28 +354,6 @@ public void setKeyboardAvailable (boolean available) {
 		this.keyboardAvailable = available;
 	}
 
-	@Override
-	public synchronized boolean isKeyPressed (int key) {
```

Comment:
Is it safe to remove this `synchronized`?

Reply:
In my opinion it is. The method only reads a field from a fixed-size array and considering that it should only get called from the main thread, there should be no simulatneous calls.